include/sparrow/data_traits.hpp

```diff
@@ -114,15 +114,15 @@ namespace sparrow
     {
         static constexpr data_type type_id = data_type::STRING;
         using value_type = std::string;
-        using default_layout = variable_size_binary_layout<value_type, std::string_view, std::string_view>;  // FIXME: this is incorrect, change when we have the right types
+        using default_layout = variable_size_binary_layout<value_type, std::string_view, const std::string_view>;  // FIXME: this is incorrect, change when we have the right types
```

Comment:
Is the FIXME comment still true or is this the final type for the read-only view?
(same question below)

Reply:
I don't know. I had to fix this to pass my UT. Where is the discussion about that topic ?
@JohanMabille any idea ?

Reply:
There: https://github.com/xtensor-stack/sparrow/pull/49/files#r1542484372

Reply:
Thanks for this link. So let's keep the comment to change the type later.